langchain/vectorstores/opensearch_vector_search.py

```diff
@@ -442,7 +471,7 @@ def from_texts(
             to "text".
 
         Optional Keyword Args for Approximate Search:
-            engine: "nmslib", "faiss", "hnsw"; default: "nmslib"
```

Comment:
is hnsw not a valid arg (just checking, not familiar with opensearch)

Reply:
seems like hnsw is the method name not the engine?

Reply:
Yes, hnsw is a method name